src/ad-products/video/jwplayer/helpers/vast-tagless-request.ts

```diff
@@ -80,7 +80,12 @@ export class VastTaglessRequest {
 			.fetch(vastUrl, this.timeout)
 			.then((res) => res.text())
 			.then((text) => this.handleTaglessResponse(text))
+			.then((vastData) => {
+				videoDisplayTakeoverSynchronizer.resolve(vastData?.lineItemId, vastData?.creativeId);
```

Comment:
Both the `lineItemId` and `creativeId` properties are always present, though their values can be `undefined`.

```suggestion
				videoDisplayTakeoverSynchronizer.resolve(vastData.lineItemId, vastData.creativeId);
```